pymc/tests/distributions/test_multivariate.py

```diff
@@ -1874,3 +1875,28 @@ def test_car_rng_fn(sparse):
         )
         f -= 1
     assert p > delta
+
+
+def test_posdef_symmetric1():
```

Comment:
I know youre just restructuring these so this isnt required, just commenting. Would be nice to paramterizing these